metacat-main/src/main/java/com/netflix/metacat/main/services/notifications/sns/SNSNotificationServiceUtil.java

```diff
@@ -140,9 +165,15 @@ public TablePartitionsUpdatePayload createTablePartitionsUpdatePayload(
      */
     public List<String> getSortedDeletionPartitionKeys(final List<PartitionDto> partitionDtos,
```

Comment:
Make this method access private if it is not being used elsewhere. Also remove the null check.

Reply:
The reason to make it public is to setup unit testing around it

Reply:
Annotate the method with VisibleForTesting

Reply:
got it.